README.md

```diff
@@ -193,4 +193,4 @@ I would like to thank Microverse for inspiring students doing such great things.
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] Kindly fix the broken link to your issues page in the readme file for professional documentation of your project.
![image](https://github.com/tawakuliKH/To_Do_List/assets/81932478/87c9f733-14a2-47f3-b9e7-016445584b52)

- [ ] Also fix the link to your LinkedIn profile as it is a broken link as well
![image](https://github.com/tawakuliKH/To_Do_List/assets/81932478/5d9d6365-e1ae-4d9d-8d2c-a8edb3fffdef)